packages/mobile/src/components/stripe-onramp-embed/StripeOnrampEmbed.tsx

```diff
@@ -0,0 +1,85 @@
+import { useCallback } from 'react'
+
+import { StyleSheet, View } from 'react-native'
+import { WebView } from 'react-native-webview'
+
+import { useIsUSDCEnabled } from 'app/hooks/useIsUSDCEnabled'
+import { useRoute } from 'app/hooks/useRoute'
+import { env } from 'app/services/env'
+
+const STRIPE_PUBLISHABLE_KEY = env.REACT_APP_STRIPE_CLIENT_PUBLISHABLE_KEY
+
+const styles = StyleSheet.create({
+  root: {
+    height: '100%',
+    width: '100%'
+  }
+})
+
+export const StripeOnrampEmbed = () => {
+  const { params } = useRoute<'StripeOnrampEmbed'>()
+  const { clientSecret } = params
+  const isUSDCEnabled = useIsUSDCEnabled()
+
+  const handleSessionUpdate = useCallback((event) => {
+    if (event?.payload?.session?.status) {
+      console.log(`Stripe Session Update ${event.payload.session.status}`)
+    }
+  }, [])
+
+  const html = `
```

Comment:
I don't suppose there's a way we could put this into an actual html file and import it? So we get a better editing experience.

Reply:
spoke with @rickyrombo I think we'd have to host the file somewhere. Totally agree but just think it's not worth the hassle.

Reply:
I think this is also doable actually!
https://stackoverflow.com/questions/42851296/react-native-load-local-html-file-into-webview

Reply:
omg!

Reply:
seeing ppl in the comments complaining that this doesn't work on android. just tried testing on android but build failed :(. would like to merge this for now and create a ticket to follow-on + test on android.

Reply:
https://linear.app/audius/issue/PAY-1663/rn-webview-load-stripe-html-from-separate-file

Reply:
yeah low pri, I don't think we're going to be iterating on this much.

Reply:
FWIW, I wasn't imaging bundling the file and loading it from an external location at runtime. I had thought we could just use our build tooling to embed the HTML into the module at build time. It was more a developer experience suggestion than anything as having a separate HTML file makes it easier to maintain. But this is fine for now!